test/e2e/testdata/wolf-rbac/cmd.sh

```diff
@@ -21,6 +21,11 @@ cd test/e2e/testdata/wolf-rbac/
 
 OPTION=$1
 
+if  ! command -v "docker-compose" > /dev/null; then
+	echo "docker-compose not found, please install docker-compose"
```

Comment:
I don't think it should be resolved that way. We should write more judgments, because my current situation is that I only have the `docker compose` command, not `docker-compose`, and I just need to replace `docker-compose` with `docker compose`.

Reply:
If so, we need to write both docker compose and docker-compose logic, which would be too cumbersome, and I noticed that docker compose is only integrated in docker desktop for Windows and Mac OS, not for Linux systems. 

 Ref https://github.com/docker/compose#windows-and-macos

Reply:
It is not necessary to ah, define a global variable. If `docker-compose` exists, copy `docker-compose` to this variable. If not, continue to determine if `docker compose` exists, and if it does, assign `docker compose` to this variable. 

Finally, replace the original `docker-compose` globally.

Reply:
> 

agree